woocommerce/woocommerce
# [Product Blocks editor]: set max width for the description general field

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



This PR sets a max-width for the description field.

### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/41335



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Product editor / General section
2. Before (trunk) confirm the visual issue reported https://github.com/woocommerce/woocommerce/issues/4133
![image](https://github.com/woocommerce/woocommerce/assets/77539/351bb32f-0121-4e18-a76a-6a100d8012db)
3. After (This PR) Confirm the description doesn't extend more than the editor width
<img width="712" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/fefe085f-91c1-418e-9f0e-d769e28edb00">



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
